### PR TITLE
Fix Portuguese locale capitalization

### PR DESCRIPTION
**Pull request by pedroma**
## Pull Request Checklist

Thank you for taking the time to improve Arrow! Before submitting your pull request, please check all *appropriate* boxes:

- [ ] 🧪 Added **tests** for changed code.
- [x] 🛠️ All tests **pass** when run locally (run `tox` or `make test` to find out!).
- [x] 🧹 All linting checks **pass** when run locally (run `tox -e lint` or `make lint` to find out!).
- [ ] 📚 Updated **documentation** for changed code.
- [x] ⏩ Code is **up-to-date** with the `master` branch.

If you have *any* questions about your code changes or any of the points above, please submit your questions along with the pull request and we will try our best to help!

## Description of Changes

This PR fixes the Portuguese locales values. They should essentially be the same as the Brasilian ones. Also fixed capitalization. We use month name with caps in Portugal.

Thanks.

### CODE REVIEW

**Comment by systemcatch** [comment]
Hi @pedroma, thank you for the PR. I see that there are similar issues with `day_names` and `day_abbreviations`, maybe you can take a look at them as well. The Brazilian locale inherits from the Portuguese locale so there should be no need for duplication for items that are identical.

**Comment by pedroma** [comment]
Hey @systemcatch. No problem. I'll look into it right now.

**Comment by pedroma** [comment]
Updated the locale. Let me know what you think.

File: arrow/locales.py [hunk 1]
```diff
@@ -2087,6 +2018,12 @@ class BrazilianPortugueseLocale(PortugueseLocale):
     day_abbreviations = ["", "Seg", "Ter", "Qua", "Qui", "Sex", "Sab", "Dom"]
 
 
+class BrazilianPortugueseLocale(PortugueseLocale):
+    names = ["pt_br"]
+
+    past = "faz {0}"
```
Comment (systemcatch) [inline arrow/locales.py:2024]: Great, if we just add a test for this one difference we should be good to go.

Comment (pedroma) [reply arrow/locales.py:2024]: Done.

**Comment by systemcatch** [comment]
Hey @pedroma  sorry I don't see the test?

**Comment by pedroma** [comment]
So sorry. I forgot to push 😄 . All done now.

**Pull request merged**